src/main/java/com/novi/TechItEasy/service/RemoteControlService.java

```diff
@@ -0,0 +1,67 @@
+package com.novi.TechItEasy.service;
+
+import com.novi.TechItEasy.dto.RemoteControlDto;
+import com.novi.TechItEasy.model.RemoteControl;
+import com.novi.TechItEasy.repository.RemoteControlRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class RemoteControlService {
+
+    private final RemoteControlRepository remoteControlRepository;
+
+    public RemoteControlService(RemoteControlRepository remoteControlRepository) {
+        this.remoteControlRepository = remoteControlRepository;
+    }
+
+    public List<RemoteControlDto> getRemoteControllers() {
+        List<RemoteControl> remoteControls= remoteControlRepository.findAll();
+        List<RemoteControlDto> remoteControlDtos = new ArrayList<>();
+
+        for (RemoteControl remoteControl : remoteControls) {
+            RemoteControlDto remoteControlDto = convertRemoteControlToRemoteControlDto(remoteControl);
+            remoteControlDtos.add(remoteControlDto);
+        }
+        return remoteControlDtos;
+    }
+
+    public RemoteControlDto createRemoteControl (RemoteControlDto remoteControlDto) {
+                remoteControlRepository.save(convertRemoteControlDtoToRemoteControl(remoteControlDto));
+                return remoteControlDto;
+    }
+
+//    -------------CONVERSION-----------
+
+    public RemoteControlDto convertRemoteControlToRemoteControlDto(RemoteControl remoteControlData) {
+        RemoteControlDto remoteControlDto = new RemoteControlDto();
+
+//  should we setId as well?
```

Comment:
Yes, is you want to use it as output. The user needs to know the id is it wants to get or delete one remotecontroller by id